docs/getting_started.md

```diff
@@ -165,7 +176,11 @@ results = mesa.batch_run(
 The results are returned as a list of dictionaries, which can be easily converted to a pandas DataFrame for further analysis.
 
 ### Visualization
-Mesa now uses a new browser-based visualization system called SolaraViz. This allows for interactive, customizable visualizations of your models. Here's a basic example of how to set up a visualization:
+Mesa now uses a new browser-based visualization system called SolaraViz. This allows for interactive, customizable visualizations of your models.
+
+The core functionality for building your own visualizations resides in the [`mesa.visualization`](apis/visualization) namespace
+
+Here's a basic example of how to set up a visualization:
 
 > **Note:** SolaraViz is experimental and still in active development for Mesa 3.0. While we attempt to minimize them, there might be API breaking changes between Mesa 3.0 and 3.1. There won't be breaking changes between Mesa 3.0.x patch releases.
```

Comment:
This note is now placed a bit unfortunate. I would move it a few lines up, maybe directly after the first paragraph.

Reply:
I am fine with moving it, but I prefer notes at the end of a section (so multiple paragraphs). Also, this is how its rendered, so the note is directly there when looking at the page.

![Screenshot 2024-11-08 at 12 49 04](https://github.com/user-attachments/assets/f64a39cf-1272-4c2b-9d62-cfe0149da447)

Reply:
It’s more that it now says: here’s an example and then you get a note, not the example.